CHANGELOG.md

```diff
@@ -9,9 +9,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Revalidate token when dashboard is reloaded or opened from different tab.
 ([#375](https://github.com/astarte-platform/astarte-dashboard/issues/375))
 - Improve history navigation by replacing the browser's location.
-([#385](https://github.com/astarte-platform/astarte-dashboard/issues/385))
+([#385](https://github.com/astarte-platform/astarte-dashboard/issues/385)) 
 - Show error metadata in live events when AstarteDeviceErrorEvent is received.
```

Comment:
there is an unwanted extra space added
Suggestion: add changes to a commit with `git add -p` instead of just adding the whole file